core/src/main/java/org/mule/expression/transformers/AbstractExpressionTransformer.java

```diff
@@ -70,7 +73,11 @@ public void initialise() throws InitialisationException
         {
             throw new InitialisationException(CoreMessages.objectIsNull("arguments[]"), this);
         }
+    }
 
+    @Override
+    public void start() throws MuleException
```

Comment:
initialise was splitted to fix issue with non initialised dependencies speculation. Will evaluate if still now that "dependency tree" has been implemented